components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1651,7 +1678,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
           qr(i,k) = qr(i,k) + (qcacc+qcaut+qrcon-qrevp)*dt
 
           if (log_predictNc) then
-             nc(i,k) = nc(i,k) + (-ncacc-ncautc+ncslf+ncnuc)*dt
+             nc(i,k) = nc(i,k) + (-ncacc-ncautc+ncslf)*dt
```

Comment:
Similarly, this removal of ncnuc isn't in PR #72 .

Reply:
This was also in response to Peter C's comment on PR #72 about where nucleation should occur.